### PR TITLE
Add 4th applying executor to bulkscan case creation

### PR DESCRIPTION
**Pull request by itoulou**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-479

### Change description ###
As part of Bulk Scan automation case creation regression testing(DTSPb-154), tech team has identified that Applying Executors for Grant or Probate (both Sols & Citizen) are restricted to 3 only, but as per the PA1P application there are four sections provided to include the applying executors names. Which ideally should have 4 executors applying details allowance while creating case in CCD by Case Worker.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/probate/service/exceptionrecord/mapper/OCRFieldAdditionalExecutorsApplyingMapperTest.java [hunk 1]
```diff
@@ -72,6 +87,68 @@ public void setUpClass() throws Exception {
...
+                .executorsApplying2applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .executorsApplying3applyingExecutorName(EXECUTOR_APPLYING_4_NAME)
+                .executorsApplying3applyingExecutorOtherNames(EXECUTOR_APPLYING_4_OTHER_NAMES)
+                .executorsApplying3applyingExecutorEmail(EXECUTOR_APPLYING_4_EMAIL)
+                .executorsApplying3applyingExecutorAddressLine1(EXECUTOR_APPLYING_4_ADDRESS_LINE1)
+                .executorsApplying3applyingExecutorAddressLine2(ADDRESS_LINE2)
+                .executorsApplying3applyingExecutorAddressTown(ADDRESS_POST_TOWN)
+                .executorsApplying3applyingExecutorAddressCounty(ADDRESS_COUNTY)
+                .executorsApplying3applyingExecutorAddressPostCode(ADDRESS_POST_CODE)
+                .build();
```
Comment (KleoG) [inline src/test/java/uk/gov/hmcts/probate/service/exceptionrecord/mapper/OCRFieldAdditionalExecutorsApplyingMapperTest.java:151]: what are these objects building?

File: src/functionalTest/resources/json/expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json [hunk 2]
```diff
@@ -1 +1 @@
...
...{"name":"Pat Jones","relationship":"Son","adoptedInOrOut":"in"}}],"spouseOrPartner":"Yes","applyingAsAnAttorney":"Yes","attorneyOnBehalfOfNameAndAddress":[{"value":{"name":"Neil Kinnock","address":{"AddressLine1":"Not Number 10","AddressLine2":"Seeing Red","AddressLine3":"","County":"Greater London","PostTown":"London Town","PostCode":"NW1 1LD","Country":""}}}],"mentalCapacity":"Yes","courtOfProtection":"No","epaOrLpa":"Yes","epaRegistered":"No","domicilityCountry":"","domicilityIHTCert":"Yes","willDate":"2013-06-20","willsOutsideOfUK":"Yes","willGiftUnderEighteen":"No","notifiedApplicants":"No","foreignAsset":"Yes","foreignAssetEstateValue":"2000000","primaryApplicantSecondPhoneNumber":"701231231231","applicationFeePaperForm":"21500","feeForCopiesPaperForm":"2000","totalFeePaperForm":"23500","paperPaymentMethod":"cheque","paymentReferenceNumberPaperform":"CHK-REF-123456","dateOfDivorcedCPJudicially":"2015-06-01","courtOfDecree":"Manchester","bulkScanCaseReference":"1000002","outsideUKGrantCopies":1,"childrenOverEighteenSurvived":"2","childrenUnderEighteenSurvived":"1","grandChildrenSurvivedUnderEighteen":"0","grandChildrenSurvivedOverEighteen":"2","caseType":"gop"}},"warnings":[]}
```
Comment (KleoG) [inline src/functionalTest/resources/json/expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json:1]: make sure no other tests are using this/break if you add anything to the payload

**Review by KleoG — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/probate/service/exceptionrecord/mapper/OCRFieldAdditionalExecutorsApplyingMapperTest.java (hunk 1 above)
Comment (itoulou) [reply src/test/java/uk/gov/hmcts/probate/service/exceptionrecord/mapper/OCRFieldAdditionalExecutorsApplyingMapperTest.java:151]: these are the builds for testing the 3rd and 4th applying executors

File: src/functionalTest/resources/json/expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json (hunk 2 above)
Comment (itoulou) [reply src/functionalTest/resources/json/expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json:1]: All functional tests passed and expectedBulkScanTransformExceptionRecordOutputCombCitizenPA1P.json isn't used anywhere else

**Review by KleoG — APPROVED** [review]

**Pull request closed without merging**